site_root/input/resources/examples/Task-example-task-id.json

```diff
@@ -0,0 +1,34 @@
+{
+  "resourceType": "Task",
+  "id": "example-task-id",
+  "status": "completed",
+  "intent": "order",
+  "input": [
+    {
+      "type": {
+        "coding": [
+          {
+            "code": "ResearchStudy"
```

Comment:
Since this `input` entry refers to a specimen, `"code"` should also be `"Specimen"`?

Reply:
resolved.